packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -144,9 +144,12 @@ export async function getPortalsTradeQuote(
       },
     } = portalsTradeOrderResponse
 
-    const rate = bn(buyAmountAfterFeesCryptoBaseUnit)
-      .div(input.sellAmountIncludingProtocolFeesCryptoBaseUnit)
-      .toString()
+    const rate = getRate({
```

Comment:
This was previously off - sell asset and buy asset may have different base units, we can't do that

Before:

<img width="220" alt="Screenshot 2024-07-29 at 12 41 05" src="https://github.com/user-attachments/assets/add1c015-d5ad-4086-8668-b9147fc31d66">
<img width="305" alt="Screenshot 2024-07-29 at 12 41 37" src="https://github.com/user-attachments/assets/35a83f04-f9d4-4da2-8709-2c8049b45ac7">

After:

<img width="220" alt="Screenshot 2024-07-29 at 12 41 05" src="https://github.com/user-attachments/assets/8830cd46-fcf0-4f97-9981-7ab71530f88c">
<img width="231" alt="Screenshot 2024-07-29 at 12 21 14" src="https://github.com/user-attachments/assets/8044b156-2d86-4c9f-abc8-d50c121ec4b6">

Reply:
Nice